core/pkg/nodestats/request.go

```diff
@@ -56,7 +56,7 @@ func (c *NodeHttpClient) makeRequest(method string, URL string, bearerToken stri
 	}
 
 	if bearerToken != "" {
-		request.Header.Add("Authorization", "bearer "+bearerToken)
```

Comment:
you are awesome, sparsh! think this would be a good opportunity to mock out a nodehttp client and inspect the outgoing response for correct capitalization?